lxd/storage/backend_lxd.go

```diff
@@ -331,7 +330,7 @@ func (b *lxdBackend) Update(clientType request.ClientType, newDesc string, newCo
 // warningsDelete deletes any persistent warnings for the pool.
 func (b *lxdBackend) warningsDelete() error {
 	err := b.state.DB.Cluster.Transaction(context.TODO(), func(ctx context.Context, tx *db.ClusterTx) error {
-		return cluster.DeleteWarnings(ctx, tx.Tx(), cluster.EntityType(entity.TypeStoragePool), int(b.ID()))
+		return cluster.DeleteWarnings(ctx, tx.Tx(), cluster.EntityTypeStoragePool, int(b.ID()))
```

Comment:
Can we use ` cluster.EntityTypeFromName(entityType.Name())` and avoid exporting ` cluster.EntityTypeStoragePool` and the other types?